components/brave_wallet_ui/common/reducers/wallet_reducer.ts

```diff
@@ -172,4 +179,47 @@ reducer.on(WalletActions.tokenBalancesUpdated, (state: any, payload: GetERC20Tok
   }
 })
 
+reducer.on(WalletActions.portfolioPriceHistoryUpdated, (state: any, payload: PortfolioTokenHistoryAndInfo[][]) => {
+  const history = payload.map((account) => {
+    return account.map((token) => {
+      if (formatBalance(token.token.assetBalance, token.token.asset.decimals) !== '0.0000') {
```

Comment:
This comparison looks error prone to me? Is it necessary to compare the result of `formatBalance`?

Reply:
Using `Number()` here instead now!